src/api/app/helpers/webui/markdown_helper.rb

```diff
@@ -7,6 +7,6 @@ def render_as_markdown(content)
                                            autolink: true,
                                            no_intra_emphasis: true,
                                            fenced_code_blocks: true, disable_indented_code_blocks: true)
-    ActionController::Base.helpers.sanitize(@md_parser.render(content.to_s))
+    ActionController::Base.helpers.sanitize(@md_parser.render(content.dup.to_s))
```

Comment:
we are passing a copy instead of the original version to avoid having the original one mutated